rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/StatisticsView.java

```diff
@@ -104,64 +107,44 @@ protected String getCellCSSText(ListGridRecord record, int rowNum, int colNum) {
         return root;
     }
 
+    private ListGridRecord createListGridRecord(String status, String type, ImageResource icon, int count) {
```

Comment:
Maybe it is better to have the prototype method that end with icon so both method createListGridRecord have the three first parameters.

Reply:
Good point thanks